bitnami/redmine/README.md

```diff
@@ -274,6 +274,7 @@ To configure Redmine to send email using SMTP you can set the following environm
 - `REDMINE_SMTP_HOST`: SMTP host.
 - `REDMINE_SMTP_PORT`: SMTP port.
 - `REDMINE_SMTP_USER`: SMTP account user.
+- `REDMINE_SMTP_DOMAIN`: SMTP domain.
```

Comment:
In the same vein, I think we should detail this is to be used when `$REDMINE_SMTP_USER` doesn't apply.